packages/mux-player/src/index.ts

```diff
@@ -700,24 +707,22 @@ class MuxPlayerElement extends VideoApiElement {
    * we aren't an audio player and the stream-type isn't live.
    */
   get storyboard() {
+    // If the storyboardSrc has been explicitly set, assume it should be used
```

Comment:
Intentionally changed the rules here a bit to always prefer the `storyboardSrc` if it's explicitly set to unlock e.g. `audio` themes that have storyboards.